### DISCUSSION TITLE
cannot resolve path for atomWithStorage

### DISCUSSION THREAD

**Discussion by adityarupifi** [pmndrs/jotai#1862] [Q&A]
## Summary

<img width="1361" alt="image" src="https://user-images.githubusercontent.com/107840653/229120035-f41206cf-6f44-4973-9a37-dd92fadf246c.png">

cannot resolve path when importing 
import { atomWithStorage, SyncStorage } from "jotai/vanilla/utils/atomWithStorage";

and persistence is not working

jotai version: 2.0.3
Webpack version: 5.77.0
react-scripts version: 5.0.1

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/107840653/229120742-ef14a796-2308-4307-8ac4-b966c6bd3da7.png">

**Comment by dai-shi** [comment]
Please import it from `jotai/vanilla/utils` (or `jotai/utils`).
